src/Engine/Spells/Spells.h

```diff
@@ -238,7 +238,7 @@ struct SpellBuff {
     PLAYER_SKILL_MASTERY uSkillMastery = PLAYER_SKILL_MASTERY_NONE; // 1-4, normal to grandmaster.
     uint16_t uOverlayID = 0;
     uint8_t uCaster = 0;
-    uint8_t uFlags = 0; // 0x1 => cast at grandmaster.
+    bool isGMBuff = false; // Buff was casted at grandmaster mastery
```

Comment:
Does this get used anywhere else? I wonder if we will need flags when loading from other games?

Reply:
In current OE snapshot it's used only like this.
First I wanted to delete it and use uPower for indication of GM Fly and Water Walk but we need to be comparable with vanilla so using this field is more convenient.